src/ContainerCommitModal.jsx

```diff
@@ -67,119 +50,116 @@ class ContainerCommitModal extends React.Component {
+                           onChange={value => { setNameError(""); setImageName(value) }} />
+            </FormGroup>
+
+            <FormGroup fieldId="commit-dialog-image-tag" label={_("Tag")}>
+                <TextInput id="commit-dialog-image-tag"
+                           placeholder="latest" // Do not translate
+                           value={tag}
+                           onChange={value => { setNameError(""); setTag(value) }} />
+            </FormGroup>
+
+            <FormGroup fieldId="commit-dialog-author" label={_("Author")}>
+                <TextInput id="commit-dialog-author"
+                           placeholder={_("Example, Your Name <yourname@example.com>")}
+                           value={author}
+                           onChange={setAuthor} />
+            </FormGroup>
+
+            <FormGroup fieldId="commit-dialog-command" label={_("Command")}>
+                <TextInput id="commit-dialog-command"
+                           value={command}
+                           onChange={setCommand} />
+            </FormGroup>
+
+            <FormGroup fieldId="commit-dialog-pause" label={_("Options")} isStack hasNoPaddingTop>
+                <Checkbox id="commit-dialog-pause"
+                          isChecked={pause}
+                          onChange={setPause}
+                          label={_("Pause container when creating image")} />
+                <Checkbox id="commit-dialog-docker"
+                          isChecked={useDocker}
+                          onChange={setUseDocker}
+                          description={_("Docker format is useful when sharing the image with Docker or Moby Engine")}
+                          label={_("Use legacy Docker format")} />
+            </FormGroup>
+        </Form>;
+
+    return (
+        <Modal isOpen
+                 showClose={false}
+                 position="top" variant="medium"
+                 title={_("Commit container")}
+                 description={fmt_to_fragments(_("Create a new image based on the current state of the $0 container."), <b>{container.Names}</b>)}
+                 footer={<>
+                     {dialogError && <ErrorNotification errorMessage={dialogError} errorDetail={dialogErrorDetail} onDismiss={setDialogError("")} />}
+                     <Button variant="primary"
+                             className="btn-ctr-commit"
+                             isLoading={commitInProgress && !nameError}
+                             isDisabled={commitInProgress || nameError}
+                             onClick={() => handleCommit(false)}>
+                         {_("Commit")}
+                     </Button>
+                     {nameError && <Button variant="warning"
+                             className="btn-ctr-commit-force"
+                             isLoading={commitInProgress}
+                             isDisabled={commitInProgress}
+                             onClick={() => handleCommit(true)}>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1146-20221124-132403-fa425528-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)